src/provisioner/const.py

```diff
@@ -25,3 +25,7 @@
 # 'UPGRADE_MODE_VAL' constant for default upgrade mode.
 UPGRADE_MODE_VAL = ""
 REQUIRED_EXTERNAL_SW = ['kafka', 'consul']
+DELTA_PATH = "yaml:///etc/cortx/delta.conf"
+DELTA_IDX = 'delta_index'
+CORTX_TMP_URL = "yaml://tmp/tmp.conf"
```

Comment:
Name CORTX_TMP_URL does not signify what it means. Need to be more accurate because its a constant.

Reply:
done, replaced with TMP_CORTX_CONF_URL